src/js/gpio.js

```diff
@@ -160,7 +160,7 @@ GpioPin.prototype.close = function(callback) {
     throw new Error('GPIO pin is not opened');
   }
 
-  this._binding.close(function(err) {
+  this._binding.close(function(err, value) {
```

Comment:
I'm not sure that this is the correct fix. In case of close I think there is no reason to get back a value (also what does the value contains?) (however I could be wrong). 

imho: the correct fix is to remove the `value` parameter one line below.